python/pyspark/mllib/clustering.py

```diff
@@ -40,11 +40,16 @@ class KMeansModel(Saveable, Loader):
 
     >>> data = array([0.0,0.0, 1.0,1.0, 9.0,8.0, 8.0,9.0]).reshape(4, 2)
     >>> model = KMeans.train(
-    ...     sc.parallelize(data), 2, maxIterations=10, runs=30, initializationMode="random")
+    ...     sc.parallelize(data), 2, maxIterations=10, runs=30, initializationMode="random",
+    ...                    seed=None, initializationSteps=5, epsilon=1e-4)
```

Comment:
I'm sorry about this, but hoped to add 1 more update.  Could you please set the seed to some fixed number in this and the other call to train() in this doc test?  It should be deterministic for stability.  (It looks like this was a problem before your PR too.)  Thanks!